### PR TITLE
First cut at better packaging samples

### PR DESCRIPTION
**Pull request by jasonrclark**
So for testing pre10 I made up a few different samples for packaging that better reflect some rough edges we've hit.

* Moved the existing `package_me.rb` into subdirectory to fit the new layout
* `imagine` packaging sample does image packaging, using relative paths to access it
* `star_wars` uses a subdirectory with requires into it, proving out our load paths. (Sample code from [one of my favorite talks I've ever given with Cora](http://confreaks.tv/videos/cascadiaruby2014-programming-in-the-small-teaching-my-5-year-old-ruby))
* `actively_supported` port of test app for checking when we're pulling in other gems

This also has a convenient `build-all` script to make it easier to stamp out a bunch of packages for testing when it comes to release time.

Fixes https://github.com/shoes/shoes4/issues/1437

### CODE REVIEW

File: samples/packaging/actively_supported/Gemfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+source "https://rubygems.org"
+
+gem "activesupport"
```
Comment (PragTob) [inline samples/packaging/actively_supported/Gemfile]: it's not  a biggy but... why of all gems would we pull in activesupport? ;P #noloveformonkeypatces 

(doesn't need to be "fixed" of course, just my grumpy self)

File: samples/packaging/actively_supported/actively_supported.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require 'active_support'
+require 'active_support/core_ext'
+
+Shoes.app do
+  if "".blank?
+    button "it was blank" do
+      alert "meh, that's useless"
```
Comment (PragTob) [inline samples/packaging/actively_supported/actively_supported.rb]: :grin:

File: samples/packaging/star_wars/app.rb [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+#
+# Original code paired on by Coraline and Jason Clark, 2014
+# https://youtu.be/jAnSsNga5Nk
```
Comment (PragTob) [inline samples/packaging/star_wars/app.rb]: :rocket:

File: samples/packaging/star_wars/characters/yoda.rb [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+      if (@ears.first.top > 220 && value > 0) || (@ears.first.top <= 150 && value < 0)
+        @animate_ears.stop
+        @animate_ears = nil
+      else
+        @ears.each do |ear|
+          ear.top += value
+        end
+      end
+    end
+  end
```
Comment (PragTob) [inline samples/packaging/star_wars/characters/yoda.rb]: app looks super interesting, I should run it :D

File: samples/packaging/actively_supported/Gemfile (hunk 1 above)
Comment (jasonrclark) [inline samples/packaging/actively_supported/Gemfile]: Ha... gotta give the people what they want. Plus, I get my digs in with the message it presents in response to the gem's presence :)

**Pull request merged**